pkg/apis/autopilot/doc.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2021 k0s authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package autopilot contains API Schema definitions for the
+// autopilot.k0sproject.io API group.
+package autopilot
+
+const GroupName = "autopilot.k0sproject.io"
```

Comment:
I wonder why linter is not failing with a public constant without a comment. Is it not required for doc.go?

Reply:
I don't think that there's a lint for that. There are quite a bunch of public constants without docs, e.g. in the constant package.

Reply:
[Revive](https://github.com/mgechev/revive) is enabled, it should nag about these. 🤔

Reply:
What's the [rule name](https://revive.run/docs#available-rules)? I only see a rule about package comments.

Reply:
Ah, EXC0011 is in default exclusions list and should be enabled separately for that to happen.

Reply:
Actually I think there is indeed a problem with revive. If I run golangci-lint in my machine I get some extra errors.

I filed an issue upstream in golangci-lint, I also need to investigate this more locally to see if there is a difference.
https://github.com/golangci/golangci-lint/issues/3764